extensions/amp-viewer-integration/0.1/messaging.js

```diff
@@ -14,18 +14,18 @@
  * limitations under the License.
  */
 
-
+import '../../../third_party/babel/custom-babel-helpers';
```

Comment:
This shouldn't go here. This is very important. I believe this probably belongs in the host.js file. `initLogConstructor` probably goes there as well. THen you should be able to use `dev().fine()`, etc.